client/next.config.js

```diff
@@ -1,4 +1,11 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
```

Comment:
Is the goal to get enough typing in place here that we can remove this setting? If not, do we need to create an action to remove this in future?

Reply:
Yes. The goal is to put the majority of typing in place. So, we should create an action to remove this one day.

Reply:
I've added a card to our Teams board.